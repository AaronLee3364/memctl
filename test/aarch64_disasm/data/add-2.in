fffffff0070c011c 8b143268
