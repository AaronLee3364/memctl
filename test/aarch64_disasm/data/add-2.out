fffffff0070c011c  ADD     X8, X19, X20, LSL #12
